src/components/links.js

```diff
@@ -1,39 +1,57 @@
 import React from 'react';
+import Helmet from 'react-helmet';
 
 function Links(props) {
   return (
-    <div className="Links">
-      <div className="link fadeInUp" style={{animationDelay: '0.2s'}}>
-        <h3>HELPLINE NUMBERS [by State]</h3>
-        <a href="https://www.mohfw.gov.in/coronvavirushelplinenumber.pdf">
-          https://www.mohfw.gov.in/coronvavirushelplinenumber.pdf
-        </a>
-      </div>
+    <div>
+      <Helmet>
+        <title>Official Support Helpline & Useful Links | Coronavirus</title>
+        <meta
+          name="title"
+          content="Official Support Helpline & Useful Links | Coronavirus"
```

Comment:
Hey :)

This almost slipped my eyes! Can you remove "official" from the title since we're not official and it would be misleading to the general public since that's the general consensus at this point.

Thanks!

Reply:
Fair point :) I'll remove it.